sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,138 @@
+  - engagement on project platforms that we manage
+- Encourage automation to improve productivity for contributors where it makes sense and consult with SIG Testing if automation is covering GitHub workflows.
+- Research other OSS projects and consult with their leaders about contributor experience topics to make sure we are always delivering value to our contributors.
+
+#### Cross-cutting and Externally Facing Processes
+
+We cross-cut all SIGs and WGs to deliver the following processes:
+
+- Deploying Changes:
+  When implementing policy changes we strive to balance  responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
+  - Low risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
+    - We will go to each lead, their mailing lists, slack channel, and/or their update meetings and ask for feedback and a [lazy consensus] process. We will follow up with a post to [kubernetes-dev@]googlegroups.com mailing list
+  - High risk changes impact a large number (<4) of SIGs, WGs, or repos, break existing contributor workflows, and are not easy to roll back. When implementing high risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
+    - Seek [lazy consensus] with a time box of at least 72 business hours with a GitHub issue link (or proposal if not applicable) to the following mailing lists:
+    - [kubernetes-sig-contribex@]googlegroups.com
+    - sig-leads@googlegroups.com
+    - [kubernetes-dev@]googlegroups.com with the GitHub issue link including the subject [NOTICE]: $announcement
+    - We will also announce it at the weekly Kubernetes Community Meeting on Thursdays
+- Depending on how wide of an ecosystem change this is, we may also slack, blog, tweet, and use other channels to get the word out.
+- Our standard time box is 72 business hours; however, there may be situations where we need to act quickly but the time period will always be clear and upfront.
+
+If we need funding for any reason, we approach [CNCF].
+CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
```

Comment:
```suggestion
CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initiatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
```

Reply:
fixed